packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -1328,7 +1330,7 @@ class FlutterView extends ServiceObject {
     // When this completer completes the isolate is running.
     final Completer<Null> completer = new Completer<Null>();
     final StreamSubscription<ServiceEvent> subscription =
-      (await owner.vm.vmService.onIsolateEvent).listen((ServiceEvent event) {
+      owner.vm.vmService.onIsolateEvent.listen((ServiceEvent event) {
```

Comment:
Unless I have misunderstood something fundamental, we are not waiting for the stream setup message to be sent here, so we should get the same error as before.